README.md

```diff
@@ -143,3 +200,12 @@ The collection key keeps track of the cache keys of a grouped collection of down
 unlikely event that this key format clashes with something in your project, you can change it.
 The expectation is for the string to have a user primary key inserted with `str.format`, so `{}`
 is required to be present.
+
+### `ASYNC_DOWNLOADS_WS_CHANNEL_NAME`
+Default: `"downloads"`
+
+The channel name for all shared information about download in channels cache layer.
+
+### `ASYNC_DOWNLOADS_CACHE_NAME`
```

Comment:
Would be good to add a section for all of the new exposed settings.

Reply:
c3c1419